### PR TITLE
MTM-47956-Data type spelling review

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md [hunk 1]
```diff
@@ -14,50 +14,50 @@ The alarm collection keeps track of alarms which have been raised. During offloa
...
+| timeOffset | integer |
+| timeWithOffset | timestamp |
+| lastUpdated | timestamp |
+| lastUpdatedOffset | integer |
+| lastUpdatedWithOffset | timestamp |
+| year | varchar |
+| month | varchar |
+| day | varchar |
+| severity | varchar |
+| history | Other |
```
Comment (rssag) [inline content/datahub/working-with-datahub-bundle/offloading-base-collections.md:32]: hm... looking at "Other" in this table, it seems odd. do cells always start with capital letter? but not the data types? @BeateRixen

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md [hunk 2]
```diff
@@ -14,50 +14,50 @@ The alarm collection keeps track of alarms which have been raised. During offloa
 
 | Column name | Column type
 | ---         |  ---
-| id | VARCHAR |
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/offloading-base-collections.md:17]: This must definitely stay as is as this is spelled that way.

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md (hunk 1 above)
Comment (BeateRixen) [reply content/datahub/working-with-datahub-bundle/offloading-base-collections.md:32]: I this case, it must anyway stay as is as this seems to be the proper spelling. I would not touch that.

Comment (rssag) [reply content/datahub/working-with-datahub-bundle/offloading-base-collections.md:32]: generally, do we capitalize the data type words in the beginning of a sentence though?

**Pull request closed without merging**